<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatasetView</class>
 <widget class="QGroupBox" name="DatasetView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>668</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Dataset</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="layoutForm">
     <item>
      <widget class="QFrame" name="formVerticalFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">e</string>
       </property>
       <layout class="QVBoxLayout" name="formFrame">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="qgb1_loadType">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Tipo Carga</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="typeLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <widget class="QCheckBox" name="qcb1_pathId">
               <property name="text">
                <string>Astral, archivo .fa</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="qcb2_pathFolder">
               <property name="text">
                <string>Carpeta</string>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="lay2_Inputs">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item row="0" column="0">
              <widget class="QToolButton" name="qtb1_pathId">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="qle1_pathId">
               <property name="text">
                <string>Path a archivo de secuencias</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QToolButton" name="qtb2_pathFolder">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="qle2_pathFolder">
               <property name="text">
                <string>Path a Folder con .ent</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="qgb2_sample">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Generar Muestra?</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="qlbl_size">
             <property name="text">
              <string>Tamaño Muestra</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="qsb_size">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>16777215</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QGroupBox" name="qgb1_filterSample">
             <property name="title">
              <string>Usar criterio</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>4</number>
              </property>
              <property name="rightMargin">
               <number>4</number>
              </property>
              <property name="bottomMargin">
               <number>4</number>
              </property>
              <item>
               <widget class="QGroupBox" name="qgb1_size">
                <property name="title">
                 <string>Tamaño Residuos</string>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <layout class="QFormLayout" name="formLayout_2">
                   <item row="0" column="0">
                    <widget class="QLabel" name="qlbl_minProtSize">
                     <property name="text">
                      <string>Mínimo</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QSpinBox" name="qsb_minProtSize">
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>180</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="maximum">
                      <number>16777215</number>
                     </property>
                     <property name="value">
                      <number>100</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QFormLayout" name="formLayout_4">
                   <item row="0" column="0">
                    <widget class="QLabel" name="qlbl_maxProtSize">
                     <property name="text">
                      <string>Máximo </string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QSpinBox" name="qsb_maxProtSize">
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>180</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="maximum">
                      <number>16777215</number>
                     </property>
                     <property name="value">
                      <number>600</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="qgb2_scopClass">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="title">
                 <string>Clasificación</string>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <layout class="QGridLayout" name="classFilter_layout">
                 <property name="leftMargin">
                  <number>5</number>
                 </property>
                 <property name="topMargin">
                  <number>5</number>
                 </property>
                 <property name="rightMargin">
                  <number>5</number>
                 </property>
                 <property name="bottomMargin">
                  <number>5</number>
                 </property>
                 <item row="0" column="1">
                  <widget class="QCheckBox" name="fold_check">
                   <property name="text">
                    <string>Pliegue</string>
                   </property>
                   <property name="autoExclusive">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QCheckBox" name="superFam_check">
                   <property name="text">
                    <string>S. Familia</string>
                   </property>
                   <property name="autoExclusive">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QCheckBox" name="family_check">
                   <property name="text">
                    <string>Familia</string>
                   </property>
                   <property name="autoExclusive">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QCheckBox" name="class_check">
                   <property name="text">
                    <string>Clase</string>
                   </property>
                   <property name="autoExclusive">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="qtv_proteins">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="actionsFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="sideFrame">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="sideFrame1Buttons">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="qpb1_loadDataset">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Cargar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="qpb3_resetDataset">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Resetear</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="qpb4_finishSelection">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Finalizar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="sideLoadLogger">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="progressWidget" native="true">
     <layout class="QFormLayout" name="formLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QProgressBar" name="qpb_progressBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>12</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
